views/layouts/news/newsPage.ejs

```diff
@@ -4,99 +4,162 @@
+        <div class="nextNews1Title" id="">
+          <p>Titre putaclic</p>
+        </div>
+        <div class="nextNews1Container">
+          <div class="nextNews1Content" id="">
+            <article>Lorem ipsum dolor sit amet, consectetur adipisicing elit. Voluptatum laboriosam incidunt eaque
+              esse. <br>
+              <button class="ui mini primary right labeled icon button"><i class="right arrow icon"></i> Lire plus </button>
+            </article>
+          </div>
+        </div>
+      </div>
+      <div class="nextNews2">
+          <div class="nextNews2Title" id="">
+            <p>Titre putaclic</p>
+          </div>
+          <div class="nextNews2Container">
+            <div class="nextNews2Content" id="">
+              <article>Lorem ipsum dolor sit amet, consectetur adipisicing elit. Voluptatum laboriosam incidunt eaque
+                esse. <br>
+                <button class="ui mini primary right labeled icon button"><i class="right arrow icon"></i> Lire plus </button>
+  
+              </article>
             </div>
-            <!-- ! First card ! -->
-            
-            <!-- Second card -->
-            <div class="card">
+          </div>
+        </div>
+    </div>
+  </div>
+  <div id="" class="ui items">
 
-                <div class="image">
-                    <img src="https://gloimg.gbtcdn.com/soa/gb/pdm-product-pic/Electronic/2018/09/14/goods_img_big-v1/20180914150034_12736.jpg">
-                </div>
+    <!-- First course -->
+    <div class="item">
 
-                <div class="content">
+        <div class="image">
+          <img src="https://via.placeholder.com/250x200" alt="Une image">
+        </div>
+  
+        <div class="content">
+  
+          <h3 class="header">Titre putaclic</h3>
+  
+          <p class="meta"><button class="ui mini right labeled icon button"><i class="tag icon"></i> WTF </button></p>
+  
+          <div class="description">
+  
+            <p>
+              Lorem ipsum dolor sit amet, consectetur adipisicing elit. Sed beatae reiciendis dolor fugiat magni, nemo ab
+              iste, nulla commodi, tempore quia natus expedita voluptatibus ullam explicabo. Fugiat perspiciatis
+              asperiores libero. Lorem ipsum dolor sit amet, consectetur adipisicing elit. Illum, nostrum sunt eos! Vero
+              officia excepturi voluptates animi hic, minima error qui placeat, facilis mollitia obcaecati nisi
+              consectetur tempore, eum ea?
+            </p>
+  
+          </div>
+          <button class="ui mini primary right labeled icon button"><i class="right arrow icon"></i> Lire plus </button>
+        </div>
+  
+      </div>
+    <!-- ! First course ! -->
 
-                    <div class="header">News 2</div>
+    <div class="ui divider"></div>
 
-                    <div class="meta">
-                        <span class="date">Date de la news 2</span>
-                    </div>
+    <!-- Second course -->
+    <div class="item">
```

Comment:
Tu répètes le code plusieurs fois, fais plutôt un template à part que tu `includes` plusieurs fois, avec une boucle `for`, tout ça